vtex/address-form
# Update Paraguay address rules

#### What is the purpose of this pull request?

Update Paraguay address rules for missing localities, as described in the ticket #238548 (Zendesk).

#### How should this be manually tested?

You can use the cart below to check the address-form both via Shipping Preview or Omnishipping for the updated regions.

- https://paraguay--vtexgame1.myvtex.com/checkout/cart/add/?sku=312&qty=1&seller=1&sc=2

#### Types of changes
